specs/Icon.md

```diff
@@ -447,9 +447,9 @@ Fabric uses global registration for its icons which needs a call to an initializ
 - [Icon registration utilities](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/utilities/icons.ts)
 - [Icon component](https://github.com/OfficeDev/office-ui-fabric-react/tree/master/packages/office-ui-fabric-react/src/components/Icon)
 
-#### Stardust
+#### Current Fluent
```

Comment:
"Current Fluent" ?

Reply:
There are numerous places where you just did a bulk find and replace without looking or reading the changes. Please fix all of them.

Something using Stardust as a title should read Fluent UI.

A URL with "stardust" in it should be a correct url.

Reply:
I did go through the changes when I did the replace. I see now that I missed one URL in a comment on Icon and there's a path that I changed below in a comment in getAccessibility.ts which I did debate but it made sense because `fluent-ui-react` does have an `ability-attributes` package. I've fixed both in the latest commits in addition to changing each thing to Fluent UI, removing current altogether.